bin/mark_changepoints_in_json

```diff
@@ -46,49 +148,59 @@ def main(in_files, window_size):
                     outliers = krun_data[filename]['all_outliers'][bench][index]
                 else:
                     outliers = list()
-                c_points_all, means = get_changepoints(cpt, p_exec[:], outliers)
-                # The last datum is always a changepoint, so we ignore it.
-                c_points = c_points_all[:-1]
-                changepoints[bench].append(c_points)
-                changepoint_means[bench].append(means)
-                classifications[bench].append(get_classification(p_exec, c_points,
-                                                                 means, window_size))
+                segments = get_segments(cpt, p_exec[:], steady_state, outliers,
```

Comment:
Does `get_segments` really mutate its second argument? I think it preemptively copies it, which means we can get rid of `[:]`.